apps/web/pages/checkout.vue

```diff
@@ -118,6 +131,11 @@ const handlePaymentMethodUpdate = async (paymentMethodId: number) => {
 };
 
 const order = async () => {
+  showTermsError.value = !termsAccepted.value;
```

Comment:
we should scroll down to the checkbox once we click the button. Otherwise the customer clicks on the buy button and does not notice the red invalid checkbox. We will add notifications later on.